sdk/data/azcosmos/cosmos_container.go

```diff
@@ -358,6 +358,50 @@ func (c *ContainerClient) ReadItem(
 	return newItemResponse(azResponse)
 }
 
+// PatchItem patches an item in a Cosmos container.
+// ctx - The context for the request.
+// partitionKey - The partition key for the item.
+// itemId - The id of the item to patch.
+// item - The content to be used to patch.
+// o - Options for the operation.
+func (c *ContainerClient) PatchItem(
+	ctx context.Context,
+	partitionKey PartitionKey,
+	itemId string,
+	item *PatchOptions,
```

Comment:
Based on the Azure Go guidelines, Options is used as a pattern for optional parameters (like ItemOptions at the end). 

One idea could be to change this to `patchOperations PatchOperations` (no pointer and renaming PatchOptions to PatchOperations).

@jhendrixMSFT do external contributions need to go through API review? How is that process?

Reply:
Can you please create an APIView from these changes so we can review it?

CC @JeffreyRichter

Reply:
@jhendrixMSFT The APIView is ready, I left some comments there, based on the outcome, we can finish polishing the PR and help @taiyokato complete it